index.html

```diff
@@ -17,7 +17,7 @@
         github: "WICG/manifest-incubations",
         shortName: "manifest-incubations",
         xref: {
-          specs: ["appmanifest"],
+          specs: ["appmanifest", "dom"],
```

Comment:
Note, if you `data-cite="DOM"` below, you don't need it here. Up to you which way you use it.